docs/source/datasets/index.rst

```diff
@@ -26,11 +26,16 @@ A **mailing address** represents the address a simulant uses to receive mail,
 which may be different -- for example, a PO box.
 Mailing addresses, not physical addresses, are recorded in tax filings.
 
-Note that in the small-scale simulated population that is available by default, these addresses all have their city/state/zip code set to the fictitious location of Anytown, WA 00000.  To read more about obtaining large-scale data with more realistic city, state, and zip code data, please see :ref:`Simulated populations <simulated_populations_main>`.
+Note that in the small-scale simulated population that is available by default, these addresses all have their 
+city/state/zip code set to the fictitious location of Anytown, WA 00000. This is to ensure that linking is not 
+unrealistically easy with the sample population (i.e., you cannot block on these attributes, as they are all identical). 
```

Comment:
Well technically you _can_ block, it just won't help you because you'd only have one block...

I think this is not necessarily about blocking _per se_, but I'm not sure exactly how to say it...

Reply:
I just tried a different way of putting it, lmk what you think!